draft-ietf-tls-tls13.md

```diff
@@ -2919,11 +2886,13 @@ Structure of this message:
        enum {
            rsa_sign(1), 
            dss_sign_RESERVED(2), 
-           rsa_fixed_dh(3), 
+           rsa_fixed_dh_RESERVED(3), 
            dss_fixed_dh_RESERVED(4),
            rsa_ephemeral_dh_RESERVED(5), 
            dss_ephemeral_dh_RESERVED(6),
-           fortezza_dms_RESERVED(20), (255)
+           fortezza_dms_RESERVED(20),
+           ecdsa_sign(64),
```

Comment:
The main point of having reserved fields here is to avoid duplicate assignment and we're going to be removing this entire type, so I don't see a lot of point in adding these.